src/property-filter/__tests__/utils.test.ts

```diff
@@ -108,47 +108,55 @@ describe('matchOperatorPrefix', () => {
 });
 
 describe('matchTokenValue', () => {
+  const property = filteringProperties.find(p => p.propertyKey === 'key') ?? null;
```

Comment:
`filteringProperties` is a constant object declared above. Do you need `?? null` if you can confidently say it is there?

or if it is not there, can this code be simplified?